lib/llm/src/protocols/common/preprocessor.rs

```diff
@@ -103,41 +127,15 @@ pub struct PreprocessedRequest {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub bootstrap_info: Option<BootstrapInfo>,
 
-    /// Data parallel rank for the request (used with data parallelism)
-    #[builder(default)]
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub dp_rank: Option<u32>,
-
     /// Additional arguments for extensibility
     #[builder(default)]
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub extra_args: Option<serde_json::Value>,
 
-    /// Extra fields requested to be included in the response's nvext
-    #[builder(default)]
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub extra_fields: Option<Vec<String>>,
-
-    /// Multimodal request-level metadata (mm_hash and token offsets)
-    #[builder(default)]
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    pub request_extra_info: Option<RequestExtraInfo>,
```

Comment:
@zhongdaor-nv is this request_extra_info not needed for multimodal kv routing anymore?